pandas/tests/copy_view/test_methods.py

```diff
@@ -174,6 +174,27 @@ def test_select_dtypes(using_copy_on_write):
     tm.assert_frame_equal(df, df_orig)
 
 
+def test_pop(using_copy_on_write):
+    df = DataFrame({"a": [1, 2, 3], "b": [4, 5, 6], "c": [0.1, 0.2, 0.3]})
+    df_orig = df.copy()
+    view_original = df[:]
+    result = df.pop("a")
+
+    assert np.shares_memory(result.values, get_array(view_original, "a"))
+    assert np.shares_memory(get_array(df, "b"), get_array(view_original, "b"))
+
+    if using_copy_on_write:
+        result.iloc[0] = 0
+        assert not np.shares_memory(result.values, get_array(view_original, "a"))
```

Comment:
```suggestion
        assert not np.shares_memory(result.values, get_array(view_original, "a"))
        assert df.iloc[0, 0] == 1
```

To ensure the original wasn't modified?

Reply:
Ah, no, nevermind. `pop` of course removed that column from `df`, so we don't have to check it there, and that's the wholeponit of having this `view_original` (which is already checked below that it is still equal to the original dataframe)